facia/app/controllers/front/FrontJsonFapi.scala

```diff
@@ -1,29 +1,41 @@
 package controllers.front
 
 import common.Logging
+import concurrent.{BlockingOperations, FutureSemaphore}
 import conf.Configuration
 import model.PressedPage
-import services.PressedPageService
+import play.api.libs.json.Json
+import services.S3
 import scala.concurrent.{ExecutionContext, Future}
 
 trait FrontJsonFapi extends Logging {
   lazy val stage: String = Configuration.facia.stage.toUpperCase
   val bucketLocation: String
+  val parallelJsonPresses = 16
```

Comment:
Increasing to 16 after seeing sporadic exceptions in prod with a low threshold of 8.